### ISSUE TITLE
NotFoundError (see above for traceback): Key MobilenetV2/Conv/BatchNorm/beta not found in checkpoint 	 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]

### ISSUE THREAD

**Issue by cuicuizhang1989** [tensorflow/models#4030]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**:
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **TensorFlow installed from (source or binary)**:
- **TensorFlow version (use command below)**:
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by cuicuizhang1989** [comment]
python ./deeplab/train.py --logtostderr --train_split="trainval" --model_variant="mobilenet_v2" --output_stride=16 --train_crop_size=513 --train_crop_size=513 --train_batch_size=4 --training_number_of_step=10 --fine_tune_batch_norm=false --tf_initial_checkpoint=./deeplabv3_mnv2_pascal_train_aug/model.ckpt-30000 --train_logdir=./log/ --dataset_dir=./deeplab/datasets/pascal_voc_seg/tfrecord/
when I run mobilenetv2 with deeplabv3+ this error is happened, who can help me?

**Comment by cuicuizhang1989** [comment]
I have soved this problem ,as the log does not clean.

**Comment by juanilarregui** [comment]
Hi @cuicuizhang1989,

I get the exact same error with Deeplab v3+ and Mobilenet_v2 during Evaluation. Could you please describe what was the solution you found?

Thanks in advance.

**Comment by dung38tn** [comment]
@juanilarregui 
clean your log file (tfmodel or tfslim_model) and you are good to go

**Comment by ghedalia95** [comment]
Hi @dung38tn, I'm not exactly sure how to clean the log file

**Comment by gmt710** [comment]
Maybe the config file does not match the model file.
